spec/models/equipment_item_spec.rb

```diff
@@ -1,160 +1,269 @@
 
 describe EquipmentItem, type: :model do
-  context 'validations' do
-    before(:each) do
-      @item = FactoryGirl.build(:equipment_item)
-    end
+  include ActiveSupport::Testing::TimeHelpers
 
-    it 'has a working factory' do
-      expect(@item.save).to be_truthy
-    end
+  it_behaves_like 'linkable'
 
+  describe 'basic validations' do
+    subject(:item) { FactoryGirl.build(:equipment_item) }
     it { is_expected.to validate_presence_of(:name) }
     it { is_expected.to validate_presence_of(:equipment_model) }
+  end
 
-    # 2015-11-09: we can't use the shoulda matchers test for scoped uniqueness
-    # due to the lack of a default value for notes - we could potentially
-    # rework our database schema to add a default value but it seems
-    # unnecessary at the moment
-    it 'ensures unique serials scoped to equipment model if it exists' do
-      em1 = FactoryGirl.create(:equipment_model)
-      em2 = FactoryGirl.create(:equipment_model)
-      FactoryGirl.create(:equipment_item, equipment_model: em1, serial: 'a')
-      em1_a2 = FactoryGirl.build(:equipment_item, equipment_model: em1,
-                                                  serial: 'a')
-      em1_nil1 = FactoryGirl.build(:equipment_item, equipment_model: em1,
-                                                    serial: nil)
-      em1_nil2 = FactoryGirl.build(:equipment_item, equipment_model: em1,
-                                                    serial: nil)
-      em1_blank1 = FactoryGirl.build(:equipment_item, equipment_model: em1,
-                                                      serial: '')
-      em1_blank2 = FactoryGirl.build(:equipment_item, equipment_model: em1,
-                                                      serial: '')
-      em2_a = FactoryGirl.build(:equipment_item, equipment_model: em2,
-                                                 serial: 'a')
-
-      expect(em1_a2.valid?).to be_falsey
-      expect(em1_nil1.save!).to be_truthy
-      expect(em1_nil2.valid?).to be_truthy
-      expect(em1_blank1.save!).to be_truthy
-      expect(em1_blank2.valid?).to be_truthy
-      expect(em2_a.valid?).to be_truthy
+  describe 'serial' do
+    it 'can be blank' do
+      item = FactoryGirl.build_stubbed(:equipment_item, serial: '')
+      expect(item.valid?).to be_truthy
+    end
+    it 'can be nil' do
+      item = FactoryGirl.build_stubbed(:equipment_item, serial: nil)
+      expect(item.valid?).to be_truthy
     end
+    it 'cannot be the same as another item of the same model' do
+      model = FactoryGirl.create(:equipment_model)
+      FactoryGirl.create(:equipment_item, equipment_model: model, serial: 'a')
+      item = FactoryGirl.build(:equipment_item, equipment_model: model,
+                                                serial: 'a')
+      expect(item.valid?).to be_falsey
+    end
+    it 'can be the same as another item of a different model' do
+      FactoryGirl.create(:equipment_item, serial: 'a')
```

Comment:
I wonder if it's better to explicitly specify that the two items belong to different equipment models?